runtime/centrifuge/src/migrations.rs

```diff
@@ -66,10 +66,13 @@ mod anemoy_pool {
 				return weight;
 			}
 
-			pallet_pool_system::Pool::<Runtime>::mutate(ANEMOY_POOL_ID, |details| {
-				let details = details.as_mut().unwrap();
-				details.currency = CURRENCY_ID_DOT_NATIVE_USDC;
-				log::info!("anemoy_pool::Migration: currency set to USDC ✓");
+			pallet_pool_system::Pool::<Runtime>::mutate(ANEMOY_POOL_ID, |maybe_details| {
+				if let Some(details) = maybe_details {
+					details.currency = CURRENCY_ID_DOT_NATIVE;
+					log::info!("anemoy_pool::Migration: currency set to USDC ✓");
+				} else {
+					log::warn!("anemoy_pool::Migration: Pool details empty, skipping migration");
+				}
```

Comment:
Comment: This would have bricked Catalyst due to unwrapping a `None` variant. We need to be more attentive of never unwrapping outside of tests.

Reply:
Thanks for the catch, @wischli. This is my bad but also reveals the necessity for a more formal review process and review requirements. In this case, requiring a migration on Centrifuge to be tested against Centrifuge AND Catalyst before being approved, whereas we only tested against Centrifuge.